typescript/packages/reality-capture-modeling/package.json

```diff
@@ -28,19 +28,13 @@
 	},
 	"license": "MIT",
 	"dependencies": {
-		"axios": "^1.6.0"
-	},
-	"peerDependencies": {
-		"@itwin/core-bentley": "^4.0.0",
-		"@itwin/core-common": "^4.0.0"
+		"axios": "^1.6.0",
+		"@itwin/reality-capture-common": "workspace:*",
```

Comment:
this is just used for types

Reply:
Fixed

Reply:
I got an error in the sample web app due to @itwin/geometry-core version mismatch (4.0 vs 4.2). I don't remember how I fixed the problem, but I tried a lot of things. Should I revert the changes?